server/mailet/integration-testing/src/test/java/org/apache/james/mailets/BounceIntegrationTest.java

```diff
@@ -257,8 +257,7 @@ private MailetContainer generateMailetContainerConfiguration(MailetConfiguration
     private ProcessorConfiguration transport() {
         // This processor delivers emails to BOUNCE_RECEIVER and POSTMASTER
         // Other recipients will be bouncing
-        return ProcessorConfiguration.builder()
-            .state("transport")
+        return ProcessorConfiguration.transport()
```

Comment:
it becomes obvious to me that we should provide a default state machine with the usual processors and the use would place custom servlets in it.
It would make the regular use case much more easy and we could write a documentation about this default state machine. WDYT ?

Reply:
I'd go just one step further: allow overriding only a specific processor from the default configuration...

Reply:
I:
 - Allowed overwritting default configuration to **specify specific processors**
 - Allowed including mailets of another processor into a given processor

This stays simple, and achive the composability you aim for.